functions/src/endpoint_paths/event.ts

```diff
@@ -125,4 +126,33 @@ events_app.post('/add', async (req, res) => {
             }
         }
     }
-})
\ No newline at end of file
+
+    // Check that user is not resident in the next page
+})
+
+/**
+ * Get all events available to the user
+ * 
+ * @throws 500 - Server Error
+ */
+events_app.post('/get', async (req, res) => {
+
+    try {
+        const user = await getUser(req["user"]["user_id"])
+        const event_logs = await getEvents(user)
+        res.status(APIResponse.SUCCESS_CODE).send({events:event_logs})
+    } catch (error) {
+        console.error("FAILED WITH ERROR: " + error.toString())
+        if (error instanceof TypeError) {
+            const apiResponse = APIResponse.InvalidDateFormat()
+            res.status(apiResponse.code).send(apiResponse.toJson())
+        }
```

Comment:
You dont need this because you are not parsing dates that could be in a bad format